blog/_posts/2023-05-26-scala-3.3.0-released.md

```diff
@@ -0,0 +1,159 @@
+
+In the following years, there will be new minor releases (3.4, 3.5, and so on) that can bring new backward-compatible features. We are calling them Scala Next; they are equivalent to Java's feature releases. Bug fixes and usability improvements from those releases will be back-ported and released as 3.3.x patches. Apart from those forward and backward-compatible changes, LTS will be feature frozen.
+
+The LTS model doesn't change anything in our compatibility guarantees. Projects built with all future releases will be able to depend on any library compiled with Scala 3.3.x. This is just our standard guarantee that the newer compiler can always consume the output of the older version.
+
+LTS might be a great choice for library authors who can now receive constant bug fixes and developer experience improvements without forcing the users of the library to update the compiler version in their project.
+
+You can read more about our compatibility guarantees in [the recent blog post](https://virtuslab.com/blog/the-scala-3-compatibility-story/).
+
+## What's new in Scala 3.3.0
+
+### Linting
+
+Scala 3.3.0 brings back linting to the Scala compiler.  Right now, you can enable checking for unused symbols and discarded values. More linting options will come soon in the following Scala 3.3.x releases.
+
+#### Checking for unused values
+
+There is an entire family of compiler options added in 3.3.0 for checking and reporting different kinds of unused symbols:
+
+- `-Wunused:imports` - for unused imports
+- `-Wunused:privates` - for unused local definitions
+- `-Wunused:locals`  - for unused local definitions
+- `-Wunused:explicits` - for unused explicit parameters
+- `-Wunused:implicits` - for unused implicit parameters (parameters in `using` clauses)
+- `-Wunused:params` - for all unused method parameters
+- `-Wunused:all` - for enabling all of the above lints
+
+#### Checking for discarded values
+
+Discarding non-unit values is usually the symptom of subtle mistakes. The compiler can now warn you about all discarded values saving you from bugs resulting from those hard-to-spot mistakes.
+
+In the following simplified example
+
+```Scala
+def failure: Either[String, Nothing] = Left("something broke")
+
+def failedComputation: Either[String, Unit] =
+  Right(()).map(_ => failure)
+```
+
+the programmer, by mistake, used the `map` method instead of `flatMap`. The code still compiles, but due to value discarding, behaves unexpectedly, returning `Right(())` from the `failedComputation` method. If the `-Wvalue-discard` flag is enabled, the compiler will report the warning, saving the user from a potential bug.
+
+### More consistent braceless syntax
+
+Braces around method parameters can now be replaced with a colon. This can lead to cleaner, shorter, and often more readable code in places like configuration DSLs or test case definitions.
```

Comment:
Given the controversial nature of Optional Brace, and now More Braceless syntax, it might be beneficial for both the maintainers and the readers to expand the context around this feature a bit, namely who proposed it, when, why, and how it was discussed and vetted (for example it was approved by SIP process in August 2022).